scripts/dist.py

```diff
@@ -43,7 +43,7 @@ def checkMavenVersion():
 
     above = MAVEN_VERSION_ABOVE.split(".")
     for index, v in enumerate(mvn_version):
-        if int(above[index]) > int(v):
+        if int(above[index]) < int(v):
```

Comment:
is the logic in this predicate correct?
we are comparing two numbers in the format `a.b.c` and `a'.b'.c'`. 
If `a<a'` or `a>a'` we can make a decision on whether to return `True` or `False` from this function, isn't it?
it is only when `a=a'` that we would compare `b` with `b'` , isn't it? and only if those 2 are equal we would then look at `c` and `c'`

Reply:
based on the warring info "Maven version must be MAVEN_VERSION_ABOVE or above in order to successfully compile all Web APIs in EMB", but the logic here seems incorrect, as it is to ensure that the version is below the MAVEN_VERSION_ABOVE version.

Reply:
you are right. the fix has been made in a new commit.